tests/ecmp/test_ecmp_sai_value.py

```diff
@@ -45,6 +45,22 @@
 offset_cmd = 'bcmcmd  "dump RTAG7_PORT_BASED_HASH 0 392 OFFSET_ECMP"'
 
 
+@pytest.fixture
+def enable_container_autorestart(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
```

Comment:
this one assume that the default state before this test is in disabled, but it may not true. Should we record what's the status before running this test? Enable during the test and recover after the test other than to assume the default is disabled?

Reply:
Ideally if the full test suite is running container autorestart should always be found disabled because of the pretest that runs in the beginning  `test_pretest.py::disable_container_autorestart`, unless there was some other test that enabled it but didn't erroneously disable it back.